.travis.yml

```diff
@@ -15,4 +15,8 @@ before_script:
 script:
 - export NODE_PATH=$TRAVIS_BUILD_DIR'/privly.safariextension/privly-applications/test/node_modules'
 - cd test
-- ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js
+- "if [ ${TRAVIS_PULL_REQUEST} = 'false' ]; then ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js; fi" #Run the unit tests
+- "cd ../privly.safariextension/privly-applications/test"
+- "if [ ${TRAVIS_PULL_REQUEST} = 'false' ]; then mv coverage/* coverage/coverage.lcov; fi" # Rename the coverage results file
```

Comment:
Every time the test is run, a the name of the coverage file changes according to https://github.com/privly/privly-applications/blob/develop/test/karma.conf-ci.js#L156
The move has been done to rename the output coverage file to 'coverage.lcov' so that it can later be referenced with this filename